src/plopp/widgets/cut3d.py

```diff
@@ -225,6 +225,8 @@ class TriCutTool(ipw.HBox):
     A collection of :class:`Cut3dTool` to make spatial cuts in the X, Y, and Z
     directions on a three-dimensional scatter plot.
 
+    .. deprecated:: v24.04.0
+
```

Comment:
Maybe a warning too like https://github.com/scipp/plopp/blob/c95e37596a8aa80ce2a30c35d965b5eb8268872b/src/plopp/graphics/figure.py#L11 ?

Reply:
I now turned this into a decorator. What do you think?